api/libproject/devstart.rb

```diff
@@ -713,32 +725,35 @@ def update_user_registered_status(cmd_name, args)
 def set_authority(cmd_name, *args)
   ensure_docker cmd_name, args
   op = WbOptionsParser.new(cmd_name, args)
+  op.opts.remove = "false"
```

Comment:
fwiw, when I tested in irb locally both `false` and `"false"` yielded the same outcome

Reply:
Switched to false.